explore-analyze/dashboards.md

```diff
@@ -1,5 +1,6 @@
 ---
 navigation_title: Dashboards
+description: Visualize and share insights from your Elasticsearch data using interactive panels, charts, maps, and custom filters.
```

Comment:
perhaps we use the {{es}} attribute here? Not 100% about using it in frontmatter